packages/forms-web-app/package.json

```diff
@@ -34,6 +34,7 @@
     "@ministryofjustice/frontend": "0.2.0",
     "@pins/business-rules": "file:../business-rules",
     "@pins/common": "file:../common",
+    "@planning-inspectorate/pins-clamav": "^1.0.8",
```

Comment:
Incorrect package name and version?  Should be pins-clamav-rest-client?

Reply:
Done